### PR TITLE
Quick Fixes

### PR DESCRIPTION
**Pull request by afischer**
This should quell 80% of the issues @adelq had with #4 and #5, but should raise at least 10 or 15 more (he's not going to like my alignment).

~Still TODO~ DONE: Refactor the dictionary out and turn it into tupptupps, but that's going to be a little obnoxious to do, so i'll do it in another PR.

### CODE REVIEW

File: pennathletics/athletes.py [hunk 1]
```diff
@@ -18,27 +18,25 @@ def get_roster(sport, year):
...
-    return list_to_return
-    
-
+        if potential_list[i].no == '':
+            passed_num = ''
+        else: 
+    	    passed_num = int(potential_list[i].no)
 
+        if ((name == "" or name == potential_list[i].name) & 
+            (no == -1 or passed_num == no) &
```
Comment (adelq) [inline pennathletics/athletes.py:36]: Nooooo these are bitwise operators, not good. `and` is the only boolean operator for Python

File: pennathletics/scraper.py [hunk 2]
```diff
@@ -1,150 +1,29 @@
...
+    # import pdb; pdb.set_trace()
+    r          = requests.get(ROSTER_URL.format(sportsdata.SPORTS[sport].SPID, sportsdata.SPORTS[sport].SPSID,year))
+    parsed     = BeautifulSoup(r.text, "html.parser")
+    info_table = parsed.find_all('table')[2].find_all('tr')
+
     for row in info_table:
-        data = [ row.find_all('td') for td in row ][0] # Get all table data
-        parsed = [ td.decode_contents(formatter='html').strip().replace(u'&nbsp;', '') for td in data ] #put it in lists, strip extraneous html
+        unparsed      = [row.find_all('td') for td in row][0] # Get all table data
+        parsed    = [td.decode_contents(formatter='html').strip().replace(u'&nbsp;', '') for td in unparsed] #put it in lists, strip extraneous html
```
Comment (adelq) [inline pennathletics/scraper.py:23]: PEP8

**Comment by adamdom** [comment]
Merging for merge conflict sake. Will fix later.

**Comment by afischer** [comment]
excuse mee

**Pull request merged**